shared/login/login/index.native.js

```diff
@@ -0,0 +1,100 @@
+// @flow
```

Comment:
this is really a rename of index.render.native like our new code (https://github.com/keybase/client/pull/7321/files#diff-b7d36e9e1a3349fd6ce4289bc1e10f59L1)